operations/kubernetes/Troubleshoot_Intermittent_503s.md

```diff
@@ -59,14 +72,14 @@ This error code typically indicates an issue with the authorization service (for
 Perform a rolling restart of all of the following:
 
 ```bash
-ncn-mw# kubectl rollout restart -n spire statefulset spire-server
+ncn-mw# kubectl rollout restart -n spire statefulset spire-postgres spire-server
 ncn-mw# kubectl rollout restart -n spire daemonset spire-agent request-ncn-join-token
-ncn-mw# kubectl rollout restart -n spire deployment spire-jwks spire-postgres spire-postgres-pooler
+ncn-mw# kubectl rollout restart -n spire deployment spire-jwks spire-postgres-pooler
+ncn-mw# kubectl rollout status -n spire statefulset spire-postgres
```

Comment:
This is not necessary as spire-postgres has already been restarted from the first command above.